src/settings/ConditionsSettings.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import _ from 'lodash';
```

Comment:
Could you please import just what you need from `lodash` here instead of the entire lib?